library/axi_dmac/tb/run_tb.sh

```diff
@@ -1,6 +1,7 @@
 NAME=`basename $0`
 
-if [[ ${SIMULATOR} != "modelsim" ]]; then
+# Default simulator is Icarus
+if [[ ${SIMULATOR} != "modelsim" ]] && [[ ${SIMULATOR} != "xsim" ]] ; then
```

Comment:
Maybe a case statement would be easier to follow.

Reply:
That's a good idea, thanks!